app/controllers/questions_controller.rb

```diff
@@ -33,8 +36,8 @@ def create
     # (so their karma/score improves as they create questions)
     current_user.vote_for(question)
 
-    msg = "Your question has been submitted! View it #{view_context.link_to("here", qset_path(question_params[:qset_id]))}"
-    redirect_to new_question_path, notice: msg
+    msg = %Q[Your question has been submitted! View it #{view_context.link_to("here", qset_path(question_params[:qset_id]))}].html_safe
```

Comment:
I think you don't need the `html_safe` here if you've set `flash: { html_safe: true }` in the redirect (line 40, below) because the `before_filter` checks for that `flash[:html_safe]` and, if present, explicitly calls `html_safe` on the `flash[:notice]`. The redundancy shouldn't hurt anything, however.